clusters/hosted-mgmt/hive/pools/pipelines/pipelines-ocp-4-14-0-amd64-openstack-clusterpool.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     tp.openshift.io/owner: pipelines
   platform:
     openstack:
-      cloud: psi-pipelines
+      cloud: openstack
```

Comment:
I’m guessing this one needs to be reverted. I think it’ It indicates the name of the openstack cloud you’re connecting to.